examples/naming_package_files/BUILD

```diff
@@ -104,3 +104,22 @@ debian_upload(
     # Refer to the target label, without having to know the final file name.
     package = ":a_deb_package",
 )
+
+#
+# Demonstrate how we can use a value from the command line in a package name.
+#
+# bazel build --//:label_from_command_line=bar  :tarball_that_needs_a_name
+# ls bazel-bin/foo-bar.tar
+label_from_command_line(
```

Comment:
Nit: "label" could be misconstrued in the bazel context.  So could "tag", and "stamp", which are the other synonyms.  Bleh.

Not sure of any other alternatives off the top of my head, though.

Reply:
I changed it to name_part.  That is sufficient different from label, tag, and stamp so it can't be confused with any of them. It is also sufficiently ugly that people won't cargo cult it.